### ISSUE TITLE
Warning with default condig: site-packages is in the MYPYPATH. Please remove it.

### ISSUE THREAD

**Issue by ssbarnea** [pre-commit/mirrors-mypy#18]
Running the hook with its default config triggers a warning:
```
~/.cache/pre-commit/repo92vl9ax8/py_env-python3/lib/python3.7/site-packages is in the MYPYPATH. Please remove it.
See https://mypy.readthedocs.io/en/latest/running_mypy.html#how-mypy-handles-imports for more info
```

There are obvious reasons to worry about this and the README.md file does not state much about it.

**Comment by asottile** [comment]
I cannot reproduce, could you provide more information?

**Comment by asottile** [comment]
for example:

```console
$ pre-commit  run mypy --verbose --all-files
mypy.....................................................................Passed
- hook id: mypy
- duration: 2.89s

Success: no issues found in 103 source files

```

```yaml
-   repo: https://github.com/pre-commit/mirrors-mypy
    rev: v0.770
    hooks:
    -   id: mypy
        exclude: ^testing/resources/
```

**Comment by ssbarnea** [comment]
Sorry another PBCAK issue, that error was coming from `flake8-mypy` which I added to flake8 hook and was running before the mypy one. Once I [removed flake8-mypy](https://github.com/ambv/flake8-mypy/issues/34), the hook run and worked fine.

As a note, I had to customize the config in order to make it behave the same way as mypy cli, mainly something like:
```
      - id: mypy
        # empy args needed in order to match mypy cli behavior
        args: []
        entry: mypy .
        pass_filenames: false
```

I am not sure why default settings are different as this may make hook adoption more confusing for new users, especially as these options are controllable using config file.
